1-js/02-first-steps/15-function-basics/article.md

```diff
@@ -1,40 +1,42 @@
 # Functions
 
-Quite often we need to perform a similar action in many places of the script.
+अक्सर हमें स्क्रिप्ट के कई स्थानों पर इसी तरह की कार्रवाई करने की आवश्यकता होती है।
```

Comment:
Hindi sentence is a bit diverted from the meaning of the original sentence, that's what I feel. Can be improved.

Reply:
done